packages/dotcom-ui-header/src/header.scss

```diff
@@ -14,6 +14,9 @@
   display: none;
 }
 
+#site-navigation {
+  position: relative;
+}
```

Comment:
**question:** Should this fix be required in the Origami header styles?

Reply:
I raised a PR for origami and moved the declaration there.

https://github.com/Financial-Times/origami/pull/1808/commits/59cef6c8c34a96d05f21a3a998cef174af82f549

Reply:
Thank you! Happy for this changed to be merged with a comment to say this rule can be removed once the PR is merged 👍

Reply:
https://github.com/Financial-Times/dotcom-page-kit/pull/1052/commits/acaa19f0a83baa64e9f425ec52afdfde5a280c8e